nixos/modules/services/web-apps/linkwarden.nix

```diff
@@ -0,0 +1,295 @@
+{
+  lib,
+  config,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.services.linkwarden;
+  isPostgresUnixSocket = lib.hasPrefix "/" cfg.database.host;
+
+  inherit (lib)
+    types
+    mkIf
+    mkOption
+    mkEnableOption
+    ;
+
+  commonServiceConfig = {
+    Type = "simple";
+    Restart = "on-failure";
+    RestartSec = 3;
+
+    EnvironmentFile = cfg.environmentFile;
+    StateDirectory = "linkwarden";
+    CacheDirectory = "linkwarden";
+    User = cfg.user;
```

Comment:
Is it mandatory to create a specific user? Can't we use dynamicUser instead?

Reply:
@drupol , just curious why shouldn't jellyfin use dynamicUser as well then?

https://github.com/NixOS/nixpkgs/blob/184ff7f559c2d4d57fafc06e5b383b0fa2c98f60/nixos/modules/services/misc/jellyfin.nix#L117-L119

Looking through git blame, heres the closest related discussion I found:

https://github.com/NixOS/nixpkgs/pull/59465/files#r276325450